.gitignore

```diff
@@ -6,3 +6,4 @@ server/public/
 **/reports/
 package-lock.json
 npm-shrinkwrap.json
+yarn.lock
```

Comment:
yarn.lock file should be source controlled (thus not git ignored). It allows deps to be installed with the correct version.

Reply:
already revised. Please review it, thanks.

Reply:
According to this, it hasn't been changed yet: https://github.com/CodeForSocialGood/calltocode.org/blob/cc61263ffb50552a36a43e84293d020f7dc7448a/.gitignore.